spec/features/idv/doc_auth/address_step_spec.rb

```diff
@@ -9,12 +9,6 @@
     complete_doc_auth_steps_before_address_step
   end
 
-  it 'is on the correct page' do
-    expect(page).to have_current_path(idv_address_path)
-    expect(page).to have_content(t('doc_auth.headings.address'))
-    expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_info'))
```

Comment:
Do we assert this anywhere else? (Do we care to?)

Reply:
Hmm, these tests do appear to be where these assertions are. I'm not sure how much value there is in these tests? I can try to find new homes for them but it will be side effects of other steps

Reply:
It would be nice if something somewhere would verify that the correct step is shown as "current" in the step indicator. I don't have any strong opinion if that's in a feature spec or as an incidental assertion of some other unrelated spec.